implants/lib/eldritch/src/lib.rs

```diff
@@ -19,6 +20,7 @@ use process::ProcessLibrary;
 use sys::SysLibrary;
 use assets::AssetsLibrary;
 use pivot::PivotLibrary;
+use crate::crypto::CryptoLibrary;
```

Comment:
Any reason to do:
`use crate::crypto::CryptoLibrary;`
Instead of
`use crypto::CryptoLibrary;`

Reply:
The rust crate I'm using is named `crypto`, so without this it becomes an ambiguous import. Must specify if you mean the crate module or the external crate